gqcp/tests/QCMethod/HF/UHF/QCMethod_UHF_stability_test.cpp

```diff
@@ -39,7 +39,7 @@ BOOST_AUTO_TEST_CASE(H3_stability_test) {
     const auto N_beta = molecule.numberOfElectronPairs();
 
     const GQCP::USpinOrbitalBasis<double, GQCP::GTOShell> spinor_basis {molecule, "STO-3G"};
-    const auto S = spinor_basis.overlap().alpha().parameters();
+    const auto S = spinor_basis.overlap();  // The alpha and beta overlap matrices are equal.
```

Comment:
```suggestion
    const auto S = spinor_basis.overlap();
```

I think I initially suggested this documentation in order to explain why we only took the 'alpha' overlap matrix. Since the API has changed, this isn't necessary anymore!